wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/Reactions/MessageReactionsCell.swift

```diff
@@ -73,48 +69,39 @@ final class MessageReactionsCell: UIView, ConversationMessageCell {
     // MARK: - configure Views and constraints
 
     private func configureSubviews() {
-        addSubview(reactionCollectionView)
-    }
-
-    private func configureConstraints() {
-        reactionCollectionView.translatesAutoresizingMaskIntoConstraints = false
-        self.translatesAutoresizingMaskIntoConstraints = false
-
-        NSLayoutConstraint.activate([
-            reactionCollectionView.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -conversationHorizontalMargins.right),
-            reactionCollectionView.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: conversationHorizontalMargins.left),
-            reactionCollectionView.topAnchor.constraint(equalTo: self.topAnchor, constant: collectionViewTopAnchorValue),
-            reactionCollectionView.bottomAnchor.constraint(equalTo: self.bottomAnchor)
-        ])
+        reactionsView.rowHeight = 24
+        addSubview(reactionsView)
+        reactionsView.fitIn(view: self, insets: insets)
```

Comment:
I think it should be `self.contentView` to properly auto-layout

Reply:
`self` here isn't a table view cell though, it's just a view that does get added to the content view of the actual cell.